templates/projects/tasks_browse.webpack.ejs

```diff
@@ -625,6 +625,33 @@
 
 <script src="<%= htmlWebpackPlugin.files.chunks.task_browse.entry %>"></script>
 <script type="text/javascript">
+    function autoRefresh() {
+      window.location = window.location.href;
```

Comment:
Are there any cons of not using reload method instead?
```javascript
window.location.reload();
```

Reply:
No cons, I think we can use reload method.